### PR TITLE
vf-5 - create partner user admin dashboard

### PR DESCRIPTION
**Pull request by asterns**
To test, try assigning roles in Auth0 that should allow you to see or not see dailykos or swingleft and make sure you can see it when you should, and get an unauthorized message when you shouldn't.  You'll need to log out/log back in again each time you change roles.  Also make sure assigning and unassigning users to partners in the db causes those records to show up or not in the csvs.  e.g.

`insert into user_partners (auth0_id,partner,source) values('[AUTH0ID]','swingleft','campaign');`

### CODE REVIEW

File: src/i18n.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sjforman) [inline src/i18n.json]: Ah, nice!

File: server/server.js [hunk 2]
```diff
@@ -995,16 +995,22 @@ router.route('/s/users').get(checkJwt, checkAdmin, function(req, res) {
...
+  const resource = partner.match(/^[a-zA-Z0-9_]*$/) ? `partner:${partner}` : '*.*';
+  const authorized = await security.isAuthorized({ token: req.token, permission: 'user.read', resource });
+  if (!authorized) {
+    res.status(401).json("unauthorized");
+  }
   if (req.query.count) {
     promise = Promise.all([
-      userService.getUsers({ count: true, preppedLetters: 'NOTPREPPED' }),
-      userService.getUsers({ count: true, preppedLetters: 'NOTSENT' })
+      userService.getUsers({ count: true, partner, userStatus: 'ADOPTEDSOME' }),
```
Comment (sjforman) [inline server/server.js]: For me at least, this change seems to break `userService-test.js`, `getUsers`, `should update prepped correctly`, but I couldn't figure out how to fix it.

Comment (sjforman) [reply server/server.js]: (That test calls `userService.getUsers` with the `preppedLetters: 'NOTPREPPED'` param set.)

**Comment by sjforman** [comment]
@asterns, I pushed up some changes to reorganize the page a bit, add a timestamp to the file name, and rationalize the columns in the downloaded csv. 

One thing I noticed is that, for me at least, the change to the method signature of `userService.getUsers` seems to have broken a test. I tried, but couldn't figure out how to fix it. Could you take a look? 

Everything else looks good to me — tested and the access control seems to be working as intended, which is awesome. 

👍

**Comment by asterns** [comment]
@sjforman - page looks a lot better, thanks 👍  Fixed the unit test, so should be all set.

**Pull request merged**